<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDRdaemonSinkGui</class>
 <widget class="QWidget" name="SDRdaemonSinkGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>190</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>190</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>FileSource</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_freq">
     <item>
      <layout class="QVBoxLayout" name="deviceUILayout">
       <item>
        <layout class="QHBoxLayout" name="deviceButtonsLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop generation</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrbase/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deviceRateLayout">
         <item>
          <widget class="QLabel" name="deviceRateText">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="freqLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Record center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnits">
       <property name="text">
        <string> kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="freqRightlSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_freq">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileSelectionLayout">
     <item>
      <widget class="QLabel" name="sampleRateLabel">
       <property name="text">
        <string>SR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ValueDial" name="sampleRate" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>Remote device sample rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sampleRateUnit">
       <property name="text">
        <string>S/s</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="interpLabel">
       <property name="text">
        <string>Int</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="interp">
       <property name="toolTip">
        <string>Interpolation</string>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_rate">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rateControlLayout">
     <item>
      <widget class="QDial" name="nbFECBlocks">
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="nominalNbBlocksText">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>000/000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="absTimeText">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Record time from start</string>
       </property>
       <property name="text">
        <string>2015-01-01  00:00:00.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addressLayout">
     <item>
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Addr:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Remote data connection IP address</string>
       </property>
       <property name="text">
        <string>000.000.000.000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataPortLabel">
       <property name="text">
        <string>D:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataPort">
       <property name="toolTip">
        <string>Remote data connection port</string>
       </property>
       <property name="text">
        <string>0000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="controlPortLabel">
       <property name="text">
        <string>C:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="controlPort">
       <property name="toolTip">
        <string>Remote control port</string>
       </property>
       <property name="text">
        <string>00000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout">
     <item>
      <widget class="QLabel" name="specificParmsLAbel">
       <property name="text">
        <string>Sp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="specificParms"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="padLayout">
     <item>
      <spacer name="verticaPadlSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
  <include location="../../../sdrbase/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
